app/src/main/java/com/example/hungrywolfes/ui/SearchedImagesAdapter.kt

```diff
@@ -0,0 +1,47 @@
+package com.example.hungrywolfes.ui
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.example.hungrywolfes.R
+import com.example.hungrywolfes.network.ListMealsImages
+
+class SearchedImagesAdapter : RecyclerView.Adapter<SearchedImagesAdapter.ImageViewHolder>() {
+
+    private val listDataImages: MutableList<ListMealsImages> = mutableListOf()
+    class ImageViewHolder(val view: View) : RecyclerView.ViewHolder(view) {
+        var mealsImages = view.findViewById<ImageView>(R.id.meal_image)
+        val mealDescription = view.findViewById<TextView>(R.id.meal_description)
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ImageViewHolder {
+        val layout = LayoutInflater
+            .from(parent.context)
+            .inflate(R.layout.item_searched_photos, parent, false)
+        return ImageViewHolder(layout)
+    }
+
+    override fun onBindViewHolder(holder: ImageViewHolder, position: Int) {
+        val photo = listDataImages[position]
+        val imageDescription = photo.strMeal
+        val imageUrl = photo.strMealThumb
```

Comment:
No need to create these 2 vals. You are only using them once, so please use photo.**attribute** where you use the vals.